### PR TITLE
operation: added mocker based select test

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: pkg/ovsdb/operation_test.go [hunk 1]
```diff
@@ -1,14 +1,49 @@
...
 	"github.com/stretchr/testify/assert"
 )
 
+const base = "../../tests/data/operation/"
+
+func readMapInterface(t *testing.T, filename string) *map[string]interface{} {
+	jsonFile, err := os.Open(filename)
+	assert.Nil(t, err)
+	defer jsonFile.Close()
+	byteValue, _ := ioutil.ReadAll(jsonFile)
```
Comment (roytman) [inline pkg/ovsdb/operation_test.go:19]: byteValue, err := ioutil.ReadAll(jsonFile)
assert.Nil(t, err)

File: pkg/ovsdb/operation_test.go [hunk 2]
```diff
@@ -1,14 +1,49 @@
...
+	var result map[string]interface{}
+	json.Unmarshal([]byte(byteValue), &result)
+	return &result
+}
+
+func readArrayMapString(t *testing.T, filename string) *[]map[string]string {
+	jsonFile, err := os.Open(filename)
+	assert.Nil(t, err)
+	defer jsonFile.Close()
+	byteValue, _ := ioutil.ReadAll(jsonFile)
```
Comment (roytman) [inline pkg/ovsdb/operation_test.go:29]: check error as above

File: pkg/ovsdb/operation_test.go [hunk 3]
```diff
@@ -1,14 +1,49 @@
...
+	var result []map[string]string
+	json.Unmarshal([]byte(byteValue), &result)
+	return &result
+}
+
+func readOperation(t *testing.T, filename string) *libovsdb.Operation {
+	jsonFile, err := os.Open(filename)
+	assert.Nil(t, err)
+	defer jsonFile.Close()
+	byteValue, _ := ioutil.ReadAll(jsonFile)
```
Comment (roytman) [inline pkg/ovsdb/operation_test.go:39]: error

File: pkg/ovsdb/operation_test.go (hunk 1 above)
Comment (hunchback) [reply pkg/ovsdb/operation_test.go:19]: ok

File: pkg/ovsdb/operation_test.go (hunk 2 above)
Comment (hunchback) [reply pkg/ovsdb/operation_test.go:29]: ok

File: pkg/ovsdb/operation_test.go (hunk 3 above)
Comment (hunchback) [reply pkg/ovsdb/operation_test.go:39]: ok

**Pull request merged**